README.md

```diff
@@ -64,66 +61,24 @@ This project can be made better and will evolve as StarkNet matures. Your contri
 - Add an explanation in the comments of the exercise if you feel it needs more explanation
 - Add exercises showcasing your favorite Cairo feature
 
-​
-
-## Getting ready to work
-
-### Step 1 - Clone the repo
+## Reusing this project
+- Clone the repo on your machine.
+- Set up the starknet environment following [these instructions](https://starknet.io/docs/quickstart.html#quickstart).
+- Set up your local starknet-devnet [official doc](https://shard-labs.github.io/starknet-devnet/)
+- Set up protostar build tool [official doc](https://docs.swmansion.com/protostar/)
+- Or use the best practise to setup the local dev env [github](https://github.com/starknet-edu/setup-starknet-dev-env/)
+- Test that you can compile the project.
 
-```bash
-git clone https://github.com/starknet-edu/starknet-erc20
-cd starknet-erc20
 ```
-
-### Step 2 - Set up your environment
-
-There are two ways to set up your environment on StarkNet: a local installation, or using a docker container
-
-- For Mac and Linux users, we recommend either
-- For windows users we recommand docker
-
-For a production setup instructions we wrote [this article](https://medium.com/starknet-edu/the-ultimate-starknet-dev-environment-716724aef4a7).
-
-#### Option A - Set up a local python environment
-
-- Set up the environment following [these instructions](https://starknet.io/docs/quickstart.html#quickstart)
-- Install [OpenZeppelin's cairo contracts](https://github.com/OpenZeppelin/cairo-contracts).
-
-```bash
-pip install openzeppelin-cairo-contracts
+protostar install OpenZeppelin/cairo-contracts@v0.5.1
+protostar build --cairo-path ./lib/cairo_contracts/src
 ```
 
-#### Option B - Use a dockerized environment
-
-- Linux and macos
-
-for mac m1:
-
-```bash
-alias cairo='docker run --rm -v "$PWD":"$PWD" -w "$PWD" shardlabs/cairo-cli:latest-arm'
+- Deploy everything using one script `deploy_starknet.py` to your local starknet-devnt
 ```
-
-for amd processors
-
-```bash
-alias cairo='docker run --rm -v "$PWD":"$PWD" -w "$PWD" shardlabs/cairo-cli:latest'
+  python3 deploy_starknet.py
 ```
 
-- Windows
-
-```bash
-docker run --rm -it -v ${pwd}:/work --workdir /work shardlabs/cairo-cli:latest
```

Comment:
I do not know, but is docker not necessary anymore?

Reply:
we can remove it. No docker needed.